README.md

```diff
@@ -105,6 +105,26 @@ print("Hi there")
 + This line is added.
 ```
 
+## Text orientation
+
+<div align="center"> Orient the text towards the center Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.  </div>
+
+
+
+<div align="right"> Orient text to the right Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
```

Comment:
Thank you for suggesting these changes @donaldte. Text orientation is more specific to HTML rather than Markdown. 

```suggestion
```

Reply:
Alright @zairahira  I get it thanks